spec/lib/workos/sso_spec.rb

```diff
@@ -83,23 +83,30 @@
         stub_request(:post, 'https://api.workos.com/sso/token').
           with(query: query).
           to_return(
+            headers: { 'X-Request-ID' => 'heroku-request-id' },
```

Comment:
```suggestion
            headers: { 'X-Request-ID' => 'request-id' },
```

Nit: `X-Request-ID` is currently a uuid generated by Heroku but if we moved to another platform, it'd be generated by something else. Maybe something more generic instead?